bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -321,6 +321,13 @@ spec:
                 - patch
                 - update
                 - watch
+            - apiGroups:
+              - packages.operators.coreos.com
```

Comment:
@qpdpQ 
Can you please help to align the indent for those permissions?

By the way, we could run `make generate-all` to update the manifests in `bundle` folder.